srcpkgs/quassel/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'quassel'
 pkgname=quassel
 version=0.13.0
-revision=1
+revision=3
 build_style=cmake
 configure_args="-DEMBED_DATA=ON -DWANT_QTCLIENT=ON -DWANT_MONO=ON -DWANT_CORE=ON
- -DUSE_QT5=ON"
+ -DUSE_QT5=ON -DWITH_BUNDLED_ICONS=OFF"
 hostmakedepends="pkg-config qt5-host-tools qt5-qmake"
 makedepends="qt5-tools-devel phonon-qt5-devel qca-qt5-devel zlib-devel
  libdbusmenu-qt5-devel qt5-script-devel qt5-plugin-tds
- qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-mysql qt5-plugin-odbc"
+ qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-mysql qt5-plugin-odbc
+ qt5-multimedia-devel $(vopt_if ldap libldap-devel)"
 depends="qt5-plugin-sqlite quassel-client-shared>=${version}_${revision}"
 _desc="Modern, cross-platform, distributed graphical IRC client"
 short_desc="${_desc} - standalone client"
```

Comment:
Mind fixing license please ?

Reply:
fixed to `GPL-2.0-only or GPL-3.0-only`

Reply:
don't think we have the `or` keyword defined in our Manual